src/meteors/attr/noise_tunnel.py

```diff
@@ -244,34 +192,96 @@ def perturb_input(
     def attribute(
         self,
         inputs: list[HSI] | HSI,
-        baselines: torch.Tensor,
+        baselines: list[torch.Tensor] | None = None,
         target: list[int] | int | None = None,
         additional_forward_args: Any = None,
         n_samples: int = 5,
         steps_per_batch: int = 1,
         perturbation_prob: float = 0.5,
         num_perturbed_bands: int | None = None,
+        perturbation_axis: None | tuple[int | slice] = None,
+        stdevs: tuple[float, ...] = (1.0,),
         method: str = "smoothgrad",
     ) -> torch.Tensor:
```

Comment:
If this is a subset of the explainer the `attribute` method needs to return HSIAttributes or a list of them but now it returns torch. You need to fix it.